stringfuzz/transformers/multiply.py

```diff
@@ -0,0 +1,38 @@
+'''
+Multiplying every integer literal by n and repeating
+every character in a string literal n times for some n
+'''
+
+import random
+import string
+
+from stringfuzz.ast import StringLitNode, IntLitNode
+from stringfuzz.ast_walker import ASTWalker
+from stringfuzz.parser import parse
+
+__all__ = [
+    'multiply',
+]
+
+MAX_FACTOR = 10
+
+class MultiplyTransformer(ASTWalker):
+    def __init__(self, ast, n=random.randint(2,MAX_FACTOR)):
```

Comment:
> Because this random computation happens at definition time before main gets called, the --seed and --random settings will not be respected.

Great catch! 

> Also, it happens exactly once, making n be random, but the same for all literals (maybe that's what we want though?).

Yup, it has to be the same value for all literals so that we get the SAT guarantees.